### PR TITLE
chore: cleans up template non-native dependencies

### PR DESCRIPTION
**Pull request by redreceipt**
Screen Shot 2022-02-06 at 3 55 00 PM

### CODE REVIEW

**Comment by vinnyjth** [comment]
image

File: templates/mobile/package.json [hunk 1]
```diff
@@ -121,14 +108,10 @@
     "eslint-import-resolver-babel-module": "^5.0.0-beta.0",
     "eslint-plugin-react": "^7.26.1",
     "get-dev-paths": "^0.1.0",
-    "graphql-import": "^0.7.1",
     "jest": "^26",
     "jetifier": "^1.6.4",
-    "metro-react-native-babel-preset": "^0.66.2",
-    "nodemon": "1.18.7",
-    "prop-types": "15.6.1",
+    "metro-react-native-babel-preset": "^0.67.2",
```
Comment (vinnyjth) [inline templates/mobile/package.json:113]:
```suggestion
    "metro-react-native-babel-preset": "^0.67.0",
```

**Comment by redreceipt** [comment]
oh. I picked one but I figured the yarn.lock would prevent that from showing up again. I'll adjust.

**Comment by vinnyjth** [comment]
@conrad-vanl in a nice plot twist, you'll have to _re_alias `nvm default` after this PR is merged ;)

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**